### DISCUSSION TITLE
Subdomains, Custom domains, & catch-alls

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1405] [Q&A] [answered]
New to SL. Enjoying it so far, but after looking through the forums/documentation I am struggling to understand the following as I don't see explicit explanations on SL or here on Github currently:

*I'm relatively new to aliases and the idea of using custom domains/subdomains as a primary email (coupled with aliases) so this is likely a result of my ignorance*

1. Do I have to own a custom domain in order to create a subdomain for it on SL? Given that subdomain contains the word 'domain', I would think that means I need a custom domain first given what I've read from other websites...but at the same time it appears as if I can just create a random subdomain (without buying a custom domain) in the 'Subdomain tab' (e.g. mail.simplelogin.com). I haven't tried submitting it to see what happens because on the off chance it does go through I don't want to burn one of my 5 subdomains unnecessarily. Also, without pressing 'Create', SL doesn't show me if the subdomain is available or being used by someone else (like I might see when searching for domain names on Namecheap). For example, if I created mail.simplelogin.com (without having to buy a custom domain first), is there anything stopping someone else from making the same subdomain? If multiple people can create a generic 'mail.simplelogin.com' subdomain, how would SL be able to determine who the e-mail goes to if multiple people also use the same generic prefix (e.g. newsletter@mail.simplelogin.com)? IP address? This is where it would seem that I would have to have a custom domain to create a subdomain in order for SL to have a unique email to forward to.

2. Both the custom domain FAQ and the subdomain FAQ describe the ability to essentially create aliases on the fly. If both can do it, am I missing something? Is there a benefit to using one over the other? This of course assumes that subdomains might be able to be created without me owning a custom domain based on what I'm seeing right now. I do understand that some websites don't accept subdomains and I also understand that custom domains offer unlimited aliases/onthefly aliases/catch-alls, while subdomains are limited to 5 total in SL...but if subdomains allow unlimited onthefly aliases as well (i.e. example1@mail.simplelogin.com, example2@mail.simplelogin.com, etc) I'm not understanding why a custom domain is beneficial (unless it's required to make subdomains) other than the fact websites may accept it because it isn't a subdomain, you can do a catch-all and because you can take your custom domain email pretty much anywhere to any mail client.

Thanks in advance for any insight you can offer!

**Comment by 4b15ed24f576** [comment, marked as answer]
Hi there, welcome to SimpleLogin! Here are more precisions regarding your subdomains and custom domains.

1. You can also create subdomains for SimpleLogin domains if it doesn't already exist, if so an error message should appear after you press "Create". _(see following screenshot)_

![RwrdkyzpSZjQULgY](https://user-images.githubusercontent.com/116150138/200561607-284cf756-5de5-48ac-944f-3b71723e4c71.png)

Subdomain can also be created for custom domain that you previously added to SimpleLogin.

2. For subdomains, using your own custom domain is beneficial if you don't want to face the hassle of finding a subdomain that is not already used by other SimepLogin's users.

A custom domain might also help you bypass some website restrictions when using a default SimpleLogin domains (rare).

Using your own domain will actually give a better control over the names of your subdomains and alias overall.

For any further questions feel free to create a thread on our [Sub Reddit](https://www.reddit.com/r/Simplelogin/), the community is pretty active there and will be happy to help you out :)

**Reply by ghost** [reply]
Thank you! Your answer was what I was looking for. I confirmed a subdomain can be created as a standalone SimpleLogin subdomain, or as a subdomain of a custom domain I own. For the standalone SimpleLogin subdomains, I was able to create one after trying a few that were already being used by other users. I also joined the sub reddit. I appreciate it!
